wiki/Tournaments/THOM_CUP/1/en.md

```diff
@@ -0,0 +1,188 @@
+
+  - If you dislike something, you are allowed to state your dislike -- however, do so reasonably. Speaking in absolutes/matter-of-fact tone is the fastest way for an argument to start, especially over an opinionated statement. Keep in mind that some of your fellow competitors and other users on this website will have content that will inevitably be used in the tournament, and it is best advised that you make it clear that it is your opinion so as to not be hurtful.
+
+
+- ***It is our right as tournament staff to reprimand users for intolerable behavior***.
+  - I will have sole discretion over decisions that affect your standing in the tournament; however, other members of tournament staff may forward any information to me that in any way represents uncalled for behavior towards other users or staff in my tournament. I will attempt to escalate serious situations beyond the scope of this event if need be, so please -- do everyone (and yourself) a favor and be respectful.
+
+  - Minor instances will receive warnings, and continued instances will result in forceful removal from the tournament.
+
+#### Warm-up Songs
+
+- Each player in the match will be allowed one (1) warm-up song to be played, making for two (2) warm-ups total between both players.
+
+- The warm-up must be uploaded to the osu! website and accessible by everyone prior to the tournament match.
+
+- Maps that have a beatmap listing but cannot be downloaded are not allowed for the purpose of warmup.
+
+- Bloodcat links are **not allowed**, and neither are personal links featuring the beatmap that you want downloaded.
+
+- Warm-up songs will be on FreeMod, allowing for any special modifications that you see fit for the purpose of warming up properly.
+
+- In the early stages, your Warm-up cant be longer than 4 minutes. This includes the first 2 stages of the tournament after the qualifiers.
+
+#### Bans, Rolls and Song Selection
+
+- With the exception of warm-ups, all songs played in a given round must match those that are selectable in the corresponding map pool. Maps are to be played on a restrictive FreeMod: only Hidden, Fade In, or Flashlight can be used.
+
+- Players are to ban one (1) map within the map pool that they are playing for the given round.
+
+- Players will be instructed to do a !roll command by the referee that is overseeing the match; the losing roll picks the first banned song, while the winning roll picks the first song of the match.
+
+- After the bans and rolls have concluded, the player with the winning roll instructs the referee of their song selection, and the referee will switch to that song. After that song concludes, the other player will pick a song, and this process will continue until one of the two conditions is satisfied:
+
+  1. One player has achieved enough wins in such a way that the other player cannot successfully win the round.
+
+    - The winning player moves onto the next match.
```

Comment:
Is this supposed to be under the 1. item? If so, add another space:

```diff
-    - The winning player moves onto the next match.
+     - The winning player moves onto the next match.
```
This applies to the others below if so.

Reply:
Seems to be okay on my end, I just tabbed twice in my editor and the preview looks fine, i dont know why it needs an extra space.